src/cli/remotes-wizard.js

```diff
@@ -2,22 +2,31 @@ import qrcode from 'qrcode-terminal';
 import chalk from 'chalk';
 import log from './log';
 import promisifyEvent from 'promisify-event';
+import dedent from 'dedent';
 
 export default async function (testCafe, remoteCount, showQRCode) {
     var connections = [];
 
+    const DESCRIPTION = dedent(`
+        Connecting ${remoteCount} remote browser(s)...
+        Navigate to the appropriate URL from each of the remote browsers.
+    `);
```

Comment:
This should be declared in the `if(remoteCount)` body to avoid unnecessary dedenting. Also, it's not a constant, but regular var